docs/user_guide/workloads/run_spark.md

```diff
@@ -75,6 +78,12 @@ rules:
 - apiGroups: [""]
   resources: ["configmaps"]
   verbs: ["get", "create", "delete"]
+- apiGroups: [""]
+  resources: ["services"]
+  verbs: ["get", "create", "delete"]
+- apiGroups: [""]
+  resources: ["persistentvolumeclaims"]
+  verbs: ["get", "create", "delete"]
```

Comment:
For default simple example runs we should not need this. Not even sure the `configmaps` is needed either.

Reply:
Ok, I am going to do some tests that the spark-pi executes with different authorizations. 
 I will use simple role and rolebinding instead of using clusterole and clusterrolebinding.